app/components/Header/Header.tsx

```diff
@@ -0,0 +1,23 @@
+"use client";
+import React from "react";
+import Logo from "./Logo/Logo";
+import Link from "./Menu/NavLink";
+import MenuNav from "./Menu/MenuNav";
+import { BiSearchAlt2 } from "react-icons/bi";
+
+const Header = ({ imgsrc, links, actionIcon }: any) => {
+  return (
+    <header className="flex justify-evenly  bg-darkBlue py-4">
+      <div className="flex gap-2">
+        <Logo ImgSrc={imgsrc} url="/" altText="image Not found" />
```

Comment:
url and altText should be added from props.
You can pass a single logo prop which is an object like logo :{
imgSrc: '...',
imgAlt:'....',
url:'....'
}